modules/claims_api/spec/fixtures/form_526_evss.json

```diff
@@ -43,13 +43,21 @@
         ]
```

Comment:
on line 26 of this file, the value "Air Force Reserve" is being used.
that should be "Air Force Reserves" (plural) to match the enum

Reply:
also, this fixture is missing the "standardClaim" field, which is required by modules/claims_api/config/schemas/526.json